hub/server/src/main/java/alluxio/hub/manager/process/ManagerProcessContext.java

```diff
@@ -268,6 +275,11 @@ public ManagerProcessContext(AlluxioConfiguration conf) {
             .withMaxDuration(
                     Duration.ofMillis(modConf.getMs(PropertyKey.HUB_MANAGER_REGISTER_RETRY_TIME)))
             .build());
+    try {
+      connectToHub(alluxio.hub.proto.AlluxioCluster.getDefaultInstance());
```

Comment:
never retried? what is the consequence of failing to connect since the exception is swallowed?

Reply:
nvm, i see that the heartbeat will retry